### DISCUSSION TITLE
permissions by GriefPrevention rank and be able to give rank permissions

### DISCUSSION THREAD

**Discussion by Hazugg** [RoinujNosde/SimpleClans#192] [Ideas]
be able to give other types of permissions by GriefPrevention rank

I mean that other types of permissions can be given by rank. If I want to give the clan accestrut in a place but to other players who have the rank such give them trust

Example: 
/Trust group.<clantag>.<rank>

![image](https://user-images.githubusercontent.com/31551514/113210150-8bbad300-9239-11eb-9266-3cf4087b2fb1.png)

be able to give rank permissions

I mean to place the permission so that the ranks of the clan can give rank, currently there is only so that they can see the list of ranks, but if I want to give a person access to give him a lower rank, you can not at least that is a leader , I would like if you could add a permission so that if I could

**Comment by RoinujNosde** [comment]
I don't understand your suggestion.

Try writing in Spanish, I may be able to understand it.

**Comment by Hazugg** [comment]
ohh my suggestion is to place that you can give permissions by clan rank as for example
/trust group.clantag.rank

and the other suggestion is to add the permission to be able to assign ranges.
give permission to a rank so that rank can give ranks within the clan

**Comment by Hazugg** [comment]
a permission to assign and that permission can give ranges
be able to put that permission to a rank, without needing to give it a clan leader so that it can assign ranks
![image](https://user-images.githubusercontent.com/31551514/113239743-f89d8f80-9270-11eb-8d4b-3d06dac438e9.png)

**Comment by Hazugg** [comment]
rank.set, could be

**Comment by RoinujNosde** [comment]
It's safer if only leaders have access to editing ranks.

**Comment by Hazugg** [comment]
You're right about that, but could you take my other GriefPrevention suggestion? be able to give other types of permissions by rank

Example:
/trust group.clantag.rank
/accesstrust group.clantag.rank

**Comment by Hazugg** [comment]
I think it would be very convenient, in case one range has the confidence level but the other does not, you want to give trust to that range without doing it one by one.

**Comment by RoinujNosde** [comment]
The one about GriefPrevention is interesting.

**Comment by Hazugg** [comment]
Yes, I just think that many would like it, also it would favor because as I comment like this, you would not have to give the permission to each one, if not only give the rank and you will already have the permission to that place
